web/src/ui_util.ts

```diff
@@ -258,3 +259,27 @@ export function show_left_sidebar_menu_icon(element: Element): void {
 export function hide_left_sidebar_menu_icon(): void {
     $(".left_sidebar_menu_icon_visible").removeClass("left_sidebar_menu_icon_visible");
 }
+
+export function disable_element_and_add_tooltip($element: JQuery, tooltip_text: string): void {
+    // Since disabled elements do not fire events, it is not possible to trigger
+    // tippy tooltips on disabled elements. So, as a workaround, we wrap the
+    // disabled element in a span and show the tooltip on this wrapper instead.
+    // https://atomiks.github.io/tippyjs/v6/constructor/#disabled-elements
+    $element.prop("disabled", true);
+    const $tooltip_target_wrapper = $("<span>");
+    $tooltip_target_wrapper.addClass("disabled-tooltip");
+    $tooltip_target_wrapper.attr("data-tippy-content", tooltip_text).attr("tabindex", "0");
+    $element.wrap($tooltip_target_wrapper);
```

Comment:
I don't see how this won't wrap an additional layer every time this is called.

Reply:
Yep, confirmed.

![image](https://github.com/user-attachments/assets/1019cf7a-a147-46fa-b40f-95933d59f231)

Reply:
Fixed like so:
``` diff
diff --git a/web/src/ui_util.ts b/web/src/ui_util.ts
index 485ffdb0f2..ae1d4e5757 100644
--- a/web/src/ui_util.ts
+++ b/web/src/ui_util.ts
@@ -278,6 +278,10 @@ export function disable_element_and_add_tooltip($element: JQuery, tooltip_text:
     // tippy tooltips on disabled elements. So, as a workaround, we wrap the
     // disabled element in a span and show the tooltip on this wrapper instead.
     // https://atomiks.github.io/tippyjs/v6/constructor/#disabled-elements
+    if ($element.prop("disabled")) {
+        // If already disabled, there's nothing to do.
+        return;
+    }
     $element.prop("disabled", true);
     const $tooltip_target_wrapper = $("<span>");
     $tooltip_target_wrapper.addClass("disabled-tooltip");
```

Reply:
I understand, did not think of the corner case. Thank you for the fix!